java/build.sh

```diff
@@ -17,8 +17,16 @@ cmake --build ./internal/build
 # Generate Panama FFM API bindings and update (if any of them changed)
 ./panama-bindings/generate-bindings.sh
 
+RUN_JAVA_TESTS="-DskipTests"
+for arg in "$@"
+do
+  if [ "$arg" == "--run-java-tests" ]; then
+    RUN_JAVA_TESTS=""
+  fi
+done
```

Comment:
```suggestion
MAVEN_VERIFY_ARGS=()
if ! hasArg --run-java-tests; then
  MAVEN_VERIFY_ARGS=("-DskipTests")
fi
```

I found the naming of `RUN_JAVA_TESTS` a bit confusing. Would you consider this alternative naming, that makes it clearer that it holds a set of arguments to be passed to `mvn verify`?

To make this suggestion work, copy the `hasArg` function from `build.sh` and put it near the top of this script

```shell
function hasArg {
    (( ${NUMARGS} != 0 )) && (echo " ${ARGS} " | grep -q " $1 ")
}
```

Then apply the suggestion in this comment.

And change the `mvn verify` call to the following:

```shell
mvn verify "${MAVEN_VERIFY_ARGS[@]}"
```